R/workflow.R

```diff
@@ -14,13 +17,14 @@
 ##'
 ##' # To run most reports, provide the report name (and the path if
 ##' # not running in the working directory, as is the case here):
-##' #id <- orderly::orderly_workflow("my_workflow", root = path)
-orderly_workflow <- function(name, root = NULL, locate = TRUE) {
+##' ids <- orderly::orderly_workflow("my_workflow", root = path)
+orderly_workflow <- function(name, envir = NULL, root = NULL, locate = TRUE,
```

Comment:
We should consider if we support a "draft" mode or not here. In draft mode we'd not commit and we'd pass in `use_draft = "newer"` to orderly run; that might be better for local use, but then it's not clear how often we'd want local use of this? Could push this to another ticket quite happily

Reply:
Stuck that in another task https://mrc-ide.myjetbrains.com/youtrack/issue/VIMC-3845